setup.cfg

```diff
@@ -1,18 +1,18 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, W504
 exclude =
   bin.src,
-  doc,
+  doc/*,
```

Comment:
Why exclude doc? It has at least one .py file (conf.py).

Also please consider `__init__.py` and `version.py` without the prefixes.

Reply:
I'm a bit confused about the "without the prefixes" suggestion. I took off the *.py from those in the config and it didn't seem to have any effect.

Reply:
Concretely I suggest changing `**/*/__init__.py` to `__init__.py` and `**/*/version.py` to `version.py`.